code/modules/shuttle/shuttle.dm

```diff
@@ -208,9 +208,12 @@
 
 	var/planet_dock = FALSE //var to help with escape pod landings
 
+/obj/docking_port/stationary/New()
+	. = ..()
+	SSshuttle.stationary += src //This has to be here to pre-empt ruin spawning
+
```

Comment:
This will not work if this is on a compiled-in map because it will reference a non-existent SSshuttle

Reply:
This is how it originally was back when we *did* use compiled maps, so no I'm pretty sure it should still work.

Testing it myself there are no runtimes when the map is compiled in like it would if SSshuttle were null.

Reply:
does not
work